### PR TITLE
alert

### PR DESCRIPTION
**Pull request by MiryalaAkshitha**

### CODE REVIEW

File: alert-component/styles.css [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    display:flex;
+  /*   flex-direction:column; */
+  /*   border:1px solid black; */
+    width:400px;
+    justify-content:center;
+    align-items:center;
+  margin-top:2rem;
+    padding:0.5rem;
+    border-radius:0.5rem;
+    color:white;
```
Comment (MurtazaBagwala07) [inline alert-component/styles.css:28]: instead of hard coding colors , use css variables

Comment (shrey27) [reply alert-component/styles.css:28]: You can define colors under root property in common css file.

**Review by shrey27 — APPROVED** [review]

**Comment by ramalingaa** [comment]
Try to use one unit system and you can remove commented css if not required.

**Review by ramalingaa — APPROVED** [review]

File: alert-component/styles.css (hunk 1 above)
Comment (MiryalaAkshitha) [reply alert-component/styles.css:28]: sure I'm doing it right away ,Thankyou

**Review by vinc3nati — APPROVED** [review]
Suggestion: use more variables for most used properties like colors, font-size , etc.

**Pull request merged**